README.md

```diff
@@ -1,17 +1,80 @@
-## Documentation Guidelines
+# Documentation Guidelines
 
-### 🚀 Contributing
+## 🚀 Contributing
 
-### Before Proposing Additions
+## Before Proposing Additions
 
-### Style Guidelines
+## Style Guidelines
 
-#### Code Examples
+### Code Examples
 
-#### Notices
+### Notices
 
-#### Screenshots
+### Screenshots
 
-#### Terminology
+### Terminology
+Appwrite has many services and features with many ways to express the same concepts. To avoid confusion, this section suggests a standard set of terms used across Appwrite documentation to describe features and concepts.
+#### General
+| **Term**      | **Suggested Usage.**                                                                                                                                                                                              |
+|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Appwrite      | Written as "Appwrite" not "AppWrite" or "appwrite".                      |
+| Projects      | Each Appwrite instance can have many Appwrite Projects. Use "Appwrite Projects" as a proper noun when referring to the concept, use project or projects when referring to specific projects.                      |
+| Console       | Refers to the the GUI. Can also be referred to as the "Appwrite Console" to distinguish from a Mac, Linux, or Windows machine's console.                                                                          |
+| Console Users | Refers to users that are registered to have access to the Appwrite Console. Not a proper noun, use as "a console user" or "the console user". Differenciate this clearly from users created through a Client SDK. |
+| Client SDK    | Refers to SDKs used by Web, Flutter, Android, and Apple applications. Use as a proper noun, use "a Client SDK" or "the Client SDKs", not client-side SDKs.                                                        |
+| Server SDK    | Refers to SDKs used by backend languages like Java, Node.js, or PHP. Use as a proper noun, use "a Server SDK" or "the Server SDKs", not server-side SDKs.                                                         |
+| Adaptor       | Refers to interfaces used to connect Appwrite with third party technologies. Adaptors are found for OAuth, Databases, Storage, and error logging. When referring to adaptors, use specific adaptor names, such as "an OAuth adaptor" or "a Storage adaptor" to avoid confusion.|
```

Comment:
We will stick with Adatper, Adaptor is technically correct in English, but I think less used. I won't dig into it further because it really doesn't matter :P

https://grammarist.com/spelling/adapter-adaptor/